webapp/src/widgets/valueSelector.scss

```diff
@@ -56,6 +56,12 @@
             }
         }
     }
+
+    div:nth-child(3) {
```

Comment:
This is flaky and is not how react-select expects you to change the styles, in this case the approach is to use the property `maxMenuHeight` (you can find it here: https://react-select.com/props#select-props), but for changing styles inside react select components normally you change the styles function generator that you can find in the `valueSelector.tsx` file.

You can see more information about modifying the styles in https://react-select.com/styles

Reply:
Done. I've fixed it in react-select's recommended way.